crates/ingress-http/src/handler/error.rs

```diff
@@ -82,7 +82,7 @@ impl HandlerError {
             HandlerError::BadAwakeablesPath => StatusCode::BAD_REQUEST,
             HandlerError::NotImplemented => StatusCode::NOT_IMPLEMENTED,
             HandlerError::Invocation(e) => {
-                invocation_status_code_to_http_status_code(e.code().into())
+                StatusCode::from_u16(e.code().into()).unwrap_or(StatusCode::INTERNAL_SERVER_ERROR)
```

Comment:
in this case there is no downcasting from u32 to u16, as that `e.code.into()` will simply unwrap this value https://github.com/restatedev/restate/pull/1290/files#diff-9ce8c7545122d14206df42ac7114b35be108a998c5a1ae112166d1837a6a93ffR19

In other places we perform the conversion u32 -> u16, where in case it doesn't fit, we replace it with 500: https://github.com/restatedev/restate/pull/1290/files#diff-9ce8c7545122d14206df42ac7114b35be108a998c5a1ae112166d1837a6a93ffR45-R52